docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+nightscout:
+  build: .
+  links:
+   - database
+   - broker
+  ports:
+   - "1337:1337"
+  environment:
+   - MONGO_CONNECTION=mongodb://database/nightscout
+   - API_SECRET=mylittlesecret
```

Comment:
Very Carefully? heh there are probablt a few ways i suppose we could make something to make a new docker-compose.yml based on input from a form or something not sure of the best way to do that.